### ISSUE TITLE
Tech Debt: Stand-down day

### ISSUE THREAD

**Issue by John-Holt-Tessella** [ISISComputingGroup/IBEX#3094]
As developer we should fix technical debt in a stand down day.

**Comment by AdrianPotter** [comment]
The decision at sprint planning on 12/4 is that we will use this sprint's stand down day to try and move E4 forwards. The day should happen after #3087 is complete. I will put in a provisional date, but we will have to postpone if the ticket isn't complete by then

**Comment by AdrianPotter** [comment]
1st May proposed on Slack. Awaiting replies

**Comment by AdrianPotter** [comment]
Happening on 26th

**Comment by AdrianPotter** [comment]
We did the day and fixed some tickets. Outstanding work is marked with the stand down flag

**Comment by Tom-Willemsen** [comment]
Outstanding work is being reviewed separately.
